src/demo/js/script.js

```diff
@@ -221,8 +226,11 @@ const tooltip = {
 };
 
 // refresh preview on interactions with the page
-document.addEventListener("keyup", () => preview.update(), false);
-document.addEventListener("click", () => preview.update(), false);
+const refresh = () => preview.update();
+document.addEventListener("keyup", refresh, false);
+document.addEventListener("click", refresh, false);
+const selectElements = [themeSelect, borderSelect, localeSelect, dateSelect, modeSelect];
```

Comment:
This looks like a good improvement.

Don't need to manually create a list of select menus, can just exclude the ones that don't need the event listener. Something like this might be more flexible:

```suggestion
const selectElements = [...document.querySelectorAll("select:not(#properties)")];
```

Also, I think it's better to avoid global variables wherever possible. I think this should be moved into some function. The `load` event function would probably be a good place for it.

A one-liner like is also not too bad:

```js
[...document.querySelectorAll("select:not(#properties)")].forEach((element) => {
    element.addEventListener("change", refresh, false)
});
```

In either case, I think it still makes sense to move all these addEventListener calls to a function.

Reply:
@DenverCoder1 thanks, that make sense, added within [e153f92](https://github.com/DenverCoder1/github-readme-streak-stats/pull/439/commits/e153f92425a0131f5a0a6803543e6da433716391)

Reply:
Great 👍